model-optimizer/extensions/front/tf/TFResizeToInterpolateV4.py

```diff
@@ -0,0 +1,132 @@
+                                      'ellipsis_mask': int64_array([0])
+                                      })
+
+    div_node = Div(graph, {'name': resize_name + '/Div_'}).create_node()
+
+    shape_to_float = Cast(graph, dict(dst_type=np.float32)).create_node()
+    size_to_float = Cast(graph, dict(dst_type=np.float32)).create_node()
+
+    size_to_float.out_port(0).connect(div_node.in_port(0))
+    shape_to_float.out_port(0).connect(div_node.in_port(1))
+    ss.out_port(0).connect(shape_to_float.in_port(0))
+    shape.out_port(0).connect(ss.in_port(0))
+
+    align_corners = resize.align_corners
+    half_pixel_centers = resize.half_pixel_centers
+
+    nearest_mode = 'floor'
+    if align_corners:
+        coordinate_transformation_mode = 'align_corners'
+        nearest_mode = 'round_prefer_ceil'
+    elif half_pixel_centers:
+        coordinate_transformation_mode = 'tf_half_pixel_for_nn' if interpolation_mode == 'nearest' else 'half_pixel'
+    else:
+        coordinate_transformation_mode = 'asymmetric'
+
+    interpolate4 = create_op_with_const_inputs(graph, Interpolate,
+                                               {
+                                                   3: int64_array([height_dim, width_dim])
+                                               },
+                                               {
+                                                   'name': resize_name + '/interpolate_4',
+                                                   'mode': interpolation_mode,
+                                                   'antialias': False,
+                                                   'coordinate_transformation_mode': coordinate_transformation_mode,
+                                                   'pads_begin': int64_array([0]),
+                                                   'pads_end': int64_array([0]),
+                                                   'nearest_mode': nearest_mode,
+                                                   'cube_coeff': -0.75,
+                                                   'shape_calculation_mode': 'sizes',
+                                                   'version': 'opset4',
+                                                   'in_ports_count': 4,
+                                               })
+
+    resize_input_connection = resize.in_port(0).get_connection()
+    resize_input_connection.set_destination(interpolate4.in_port(0))
+    resize_input_connection.get_source().connect(shape.in_port(0))
+
+    div_node.out_port(0).connect(interpolate4.in_port(2))
+
+    sizes_connection = resize.in_port(1).get_connection()
+    sizes_connection.set_destination(interpolate4.in_port(1))
+    sizes_connection.get_source().connect(size_to_float.in_port(0))
+
+    resize.out_port(0).get_connection().set_source(interpolate4.out_port(0))
+    rename_nodes([(resize, resize_name + '/delete_'), (interpolate4, resize_name)])
+
+
+class TFResizeToInterpolateV4(FrontReplacementOp):
```

Comment:
Let's remove "V4" from the name of a class and a file. In the future we may introduce new version of the Interpolate and this name will become invalid. The same comment for other files with similar issue.

Reply:
Done.